src/tests.js

```diff
@@ -1,4 +1,5 @@
 'use strict';
+
 const { Crypto } = require('./crypto');
```

Comment:
```suggestion
const { Crypto } = require('./crypto.js');
```
Adding extension removes additional checks inside `require`, otherwise it will search `/crypto/index.js` and `/crypto.json` files.